services/web-app/pages/assets/functions.js

```diff
@@ -33,8 +33,7 @@ const Functions = ({ librariesData }) => {
     <>
       <NextSeo {...seo} />
       <PageHeader bgColor={func.bgColor} title={seo.title} pictogram={func.icon} />
-      {/* this probably eventually gets replaced  by a functionsCatalog */}
```

Comment:
@francinelucca I didn't see the need for a separate functions catalog component here, but let me know if I missed something!

Reply:
you're right! not needed 👯